tmt/steps/__init__.py

```diff
@@ -734,6 +734,35 @@ def command(cls) -> click.Command:
                 command = option(command)
             commands[method.name] = command
 
+            for param in command.params:
+                if param.name == 'how':
+                    continue
+
+                if not isinstance(param, click.Option):
+                    continue
+
+                assert command.name is not None  # narrow type
+                assert param.name is not None  # narrow type
+
+                command_name, method_name, param_name = \
+                    command.name.upper(), method.name.upper(), param.name.upper()
+
+                envvar = f'TMT_PLUGIN_{command_name}_{method_name}_{param_name}'
+
+                if param.envvar is None:
+                    param.envvar = [envvar]
+
+                elif isinstance(param.envvar, str):
```

Comment:
This would deserve a comment, it is unclear why this is here. Is it because we expect envvar already set by click option, which would define its own environment variable?

Reply:
Fixed in e88cadc2